server/routes/place/html.py

```diff
@@ -32,6 +33,19 @@
 }
 
 
+def get_place_summaries() -> dict:
+  """Load place summary content from GCS"""
+  try:
+    tmp_path = os.path.join('/tmp', PLACE_SUMMARY_CONTENT_FILENAME)
+    current_app.config['PLACE_SUMMARY_BLOB'].download_to_filename(tmp_path)
```

Comment:
This may have race condition, when two client requests in the same time and both write to the file. If the whole setup works, later on can change this to read bytes directly.